Update dev_mailserver.md

jfoclpf opened the pull request:
Mail server configuration now is done on `config/secrets.yml` and thus documentation should be updated accordingly

jfoclpf committed:
Update dev_mailserver.md

Mail server configuration now is done on `config/secrets.yml` and thus documentation should be updated accordingly

javierm reviewed (commented):
This is awesome, thanks!

Just one thing: how about changing the title and the navigation link from "Development Mail Server" to "Mail Server Configuration"?

jfoclpf committed:
Update and rename dev_mailserver.md to mail_server_configuration.md

change the title and the navigation link from "Development Mail Server" to "Mail Server Configuration"

jfoclpf commented:
> Just one thing: how about changing the title and the navigation link from "Development Mail Server" to "Mail Server Configuration"?

Done it! Please double check!

javierm commented on english-documentation/introduction/servers/mail_server_configuration.md:
```diff
@@ -0,0 +1,42 @@
+# Mail Server Configuration
+
+This is a example to how integrate a mailing service with a development environment of Consul.
```
```suggestion
This is an example of how to integrate a mailing service with Consul.
```

javierm reviewed (commented):
@jfoclpf Sorry for the super-late review! :pray: :grimacing:

If you're busy and unable to keep working on this pull request, I'll understand. If that's the case, I'll open a new pull request with your changes, keeping you as the author.

Other than the comment I've left, we've removed the `english-documentation` folder because it had been added by mistake. So the changes you've done in this pull request should be in the `en` folder instead. Could you rebase this pull request so it points to the latest `master`?

Then, could you update the `SUMMARY.md` file?

```diff
diff --git a/SUMMARY.md b/SUMMARY.md
index 6faf98c..a84964c 100644
--- a/SUMMARY.md
+++ b/SUMMARY.md
@@ -24,7 +24,7 @@
     * [Digital Ocean](english-documentation/introduction/servers/digital_ocean.md)
     * [Heroku](spanish-documentation/introduction/servers/deploying-on-heroku.md)
     * [Docker](english-documentation/introduction/servers/docker.md)
-    * [Development Mail Server](english-documentation/introduction/servers/dev_mailserver.md)
+    * [Mail Server Configuration](en/installation/mail_server_configuration.md)
   * [Basic configuration](english-documentation/introduction/basic_configuration.md)
   * [CONSUL Documentation and guides](english-documentation/introduction/documentation_and_guides.md)
 * [Customization](english-documentation/customization/README.md)
```

Thanks a lot and, again, sorry it took that long to come back to this pull request!

javierm commented:
@jfoclpf I've applied your changes and updated the Spanish documentation as well in pull request #110. Thank you so much! :pray:

Closing in favor of #110.

javierm commented:
And, once again, very sorry we kept you waiting for a year :pray:.

jfoclpf commented:
No problem  :)

All the best

Javi Martín ***@***.***> schrieb am Mi., 10. Nov. 2021, 02:14:

> And, once again, very sorry we kept you waiting for a year 🙏.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/consul/docs/pull/101#issuecomment-964690645>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA6M4DN2OASZWMNUQ25IA4TULHBRFANCNFSM4TUQKLZQ>
> .
>
